doc/modules/mod_event_pusher_push.md

```diff
@@ -1,128 +1,59 @@
-
-## Configuration examples
-
-### XEP-0357 compliant with local PubSub
-
-This is the second most versatile setup.
-It allows your clients to enable push notifications via either a local PubSub or a remote one.
-
-```Erlang
-{mod_pubsub, [{plugins, [<<"push">>]}]}, % mandatory minimal config
+This module is very easy to enable, just paste the following to your MongooseIM configuration file:
 
-{mod_event_pusher, [
-    {backends, [
-        {push, [
-            {backend, mnesia}, % optional
-            {wpool, [{workers, 200}]}, % optional
-            {plugin_module, mod_event_pusher_push_plugin_defaults} % optional
-        ]}
-    ]}
-]}
 ```
-
-### XEP-0357 compliant with remote PubSub
-
-In this case, clients must provide a domain of a PubSub service that exists on another XMPP cluster.
-Notifications will be delivered over XMPP Federation to the remote server(s).
-
-```Erlang
 {mod_event_pusher, [
-    {backends, [
-        {push, [
-            {backend, mnesia}, % optional
-            {wpool, [{workers, 200}]}, % optional
-            {plugin_module, mod_event_pusher_push_plugin_defaults} % optional
-        ]}
-    ]}
-]}
+  {backends, [
+    {push, [{wpool, [{workers, 100}]}]}
+  ]}
+]}.
 ```
 
-### Private deployment only
+And that's basically it. You have just enabled the push notification support
+with 100 asynchronous workers that will handle all push notification related work.
 
-A direct connection to a push service (e.g. MongoosePush) must be configured on the same MongooseIM node.
-PubSub is completely bypassed and the clients don't need to create a push node.
-They only have to provide the virtual PubSub address.
 
-```Erlang
-{mod_event_pusher, [
-    {backends, [
-        {push, [
-            {backend, mnesia}, % optional
-            {wpool, [{workers, 200}]}, % optional
-            {plugin_module, mod_event_pusher_push_plugin_defaults}, % optional
-            {virtual_pubsub_hosts, ["virtual.@HOST@"]}
-        ]}
-    ]}
-]}
-```
-
-### XEP-0357 compliant + virtual hosts
-
-This is the most versatile setup. Allows the clients to use all push notifications configurations:
+## Options
 
-* Push service handled by another cluster via XEP-0357
-* Push service handled locally via XEP-0357
-* PubSub-less
+* **backend** (atom, default: `mnesia`) - Backend to use for storing the registrations.
+ Possible options are `mnesia` and `rdbms`.
+* **wpool** (list, default: `[]`) - List of options that will be passed to the `worker_pool` library that handles all the requests.
+ Please refer to the [Project Site](https://github.com/inaka/worker_pool) for more details.
+* **plugin_module** (atom, default: `mod_event_pusher_push_plugin_defaults`) - module implementing
+  `mod_event_pusher_push_plugin` behaviour, used for dynamic configuration of push notifications.
+  Read more about it [here](#plugin-module)
```

Comment:
I personally like the `relevant section` approach used in line 39 instead of `here` with a link.

Reply:
That's cool 👍